code/modules/mob/living/carbon/xenomorph/XenoProcs.dm

```diff
@@ -747,3 +747,16 @@
 /// Handler callback to reset immobilization status after a successful [/mob/living/carbon/xenomorph/proc/throw_carbon]
 /mob/living/carbon/xenomorph/proc/throw_carbon_end(mob/living/carbon/target)
 	REMOVE_TRAIT(target, TRAIT_IMMOBILIZED, XENO_THROW_TRAIT)
+
+/// snowflake proc to clear effects from research warcrimes
+/mob/living/carbon/xenomorph/proc/clear_debuffs()
```

Comment:
You also need `/datum/action/xeno_action/activable/warden_heal/use_ability` to call this when in `WARDEN_HEAL_DEBUFFS` mode (and for it to remove the set status effects that it used to do because they'll be now done here).